### PR TITLE
Add the sub-command 'instance' of command 'dependency'

### PR DESCRIPTION
**Pull request by JarvisG495**
swctl dependency instance [clientService-id] [serverService-id] [--start=start-time] [--end=end-time]

Query the instance topology based on the given clientServiceID and serverServiceID. Results are in JSON format.

Closes apache/Skywalking#7822

### CODE REVIEW

**Comment by kezhenxu94** [comment]
@wankai123 FYI

**Review by wu-sheng — DISMISSED** [review]
Command parameters look good to me. Wait for code review.

File: scripts/test_commands.sh [hunk 1]
```diff
@@ -70,4 +70,6 @@ ${swctl} --display=json db g >/dev/null 2>&1
 # Test `dependency`
 ${swctl} --display=json dependency service "test" > /dev/null 2>&1
 
-${swctl} --display=json dependency endpoint "test" > /dev/null 2>&1
\ No newline at end of file
+${swctl} --display=json dependency endpoint "test" > /dev/null 2>&1
+
+${swctl} --display=json dependency endpoint "test" > /dev/null 2>&1
```
Comment (kezhenxu94) [inline scripts/test_commands.sh:75]: Duplicate? Or copy-paste problem?

Comment (JarvisG495) [reply scripts/test_commands.sh]: My bad. It should be {swctl} --display=json dependency instance "test" "test".

**Review by kezhenxu94 — APPROVED** [review]
Thank you @JarvisG495 !!!

**Comment by wu-sheng** [comment]
@JarvisG495 When you have time and if you have interest, I hope you could enhance this CLI a little more. Add `service name`, `instance name`, `endpoint name` for metrics, dependency, trace, log query.

We have to use complex shell command to `get id first`, then do real query. The 2 side effects are
1. It is not very readable.
2. Different OS platforms(especially Linux VS Windows), they can't share the same command.

**Comment by JarvisG495** [comment]
> @JarvisG495 When you have time and if you have interest, I hope you could enhance this CLI a little more. Add `service name`, `instance name`, `endpoint name` for metrics, dependency, trace, log query.
> 
> We have to use complex shell command to `get id first`, then do real query. The 2 side effects are
> 
> 1. It is not very readable.
> 2. Different OS platforms(especially Linux VS Windows), they can't share the same command.

OK. I will try to implement that in the following weeks.
I also planned to add graphical display for 'dependency' command, but have not finished it yet.

**Comment by wu-sheng** [comment]
That is cool 😄

**Pull request merged**